src/components/Card/Card.js

```diff
@@ -1,16 +1,28 @@
 import { Tile } from 'carbon-components-react';
-import React from 'react';
+import React, { useState } from 'react';
 
-const Card = () => (
-    <Tile className='bx--tile--clickable card'>
+
+const Card = () => {
+    const uncheckedCardStyle = 'bx--tile--clickable card';
+    const checkedCardStyle = 'bx--tile--clickable card__checked';
+    
+    const [cardStyleValue, setCardStyleValue] = useState(uncheckedCardStyle);
```

Comment:
1. It's a bad practice to contain styles in state. State represents the property of object, some description of entity. In this case, If you pass definition of whether checkbox is clicked or not - you can use bool value to describe it. In that case you can simplify your code by simple setting state to !cardStyleValue, switching it between 2 bool values.
2. Even more - since you have one style set any way - you can set only additional one using ternary operator - even inline.
3. To et it all look prettier, you can use this [package](https://www.npmjs.com/package/classnames), it is often used in projects for conditional styling.